_posts/01-02-01-Use-the-Current-Stable-Version.md

```diff
@@ -1,15 +1,15 @@
 ---
-title:   Use the Current Stable Version (8.1)
+title:   Use the Current Stable Version (8.2)
 isChild: true
 anchor:  use_the_current_stable_version
 ---
 
-## Use the Current Stable Version (8.1) {#use_the_current_stable_version_title}
+## Use the Current Stable Version (8.2) {#use_the_current_stable_version_title}
 
-If you are getting started with PHP, start with the current stable release of [PHP 8.1][php-release]. PHP 8.x adds many [new features](#language_highlights) over the older 7.x and 5.x versions. The engine has been largely re-written, and PHP is now even quicker than older versions. PHP 8 is a major update of the language and contains many new features and optimizations.
+If you are getting started with PHP, start with the current stable release of [PHP 8.2][php-release]. PHP 8.x adds many [new features](#language_highlights) over the older 7.x and 5.x versions. The engine has been largely re-written, and PHP is now even quicker than older versions. PHP 8 is a major update of the language and contains many new features and optimizations.
 
 You should try to upgrade to the latest stable version quickly - PHP 7.4 [is already End of Life](http://php.net/supported-versions.php). Upgrading is easy, as there are not many [backwards compatibility breaks][php-bc]. If you are not sure which version a function or feature is in, you can check the PHP documentation on the [php.net][php-docs] website.
 
-[php-release]: http://php.net/downloads.php
-[php-docs]: http://php.net/manual/
-[php-bc]: http://php.net/manual/migration81.incompatible.php
+[php-release]: https://php.net/downloads.php
+[php-docs]: https://php.net/manual/
+[php-bc]: https://php.net/manual/migration82.incompatible.php
```

Comment:
Maybe keep the 8.1 migration link here as well? The 8.2 link only talks about migrating from 8.1, but the previous sentence talks about 7.4.